docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,8 +6,53 @@
 
 :::
 
+## V1.9.5 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.5)
+
+**Banff.5 - Warp Messaging**
+
+This version is backwards compatible to
+[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is
+optional, but encouraged. The supported plugin version is `21`.
+
+### Subnet Messaging
```

Comment:
Please replace ### with ** ** in all places. It creates header.

Reply:
Resolved